sim-cli/src/parsing.rs

```diff
@@ -140,23 +190,94 @@ impl TryFrom<&Cli> for SimulationCfg {
     }
 }
 
+struct NodeMapping {
+    ///Tracks nodes-info by pubkey
+    pk_node_map: HashMap<PublicKey, NodeInfo>,
+    ///Tracks nodes-info by aliases
+    alias_node_map: HashMap<String, NodeInfo>,
+}
```

Comment:
nit: missing space after `//` and needs full stop or delete the comment, I think the field names are quite self-explanatory